### ISSUE TITLE
Persistent non-visualisation of the fetal stomach

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7373]
**Preferred term label:**
Persistent non-visualisation of the fetal stomach

QUESTION -- is this the same as [Absence of stomach bubble on fetal sonography HP:0010963](https://hpo.jax.org/app/browse/term/HP:0010963) and if so should we add the synonyms below?

**Synonyms**
Absent stomach
Absent fetal stomach
Collapsed fetal stomach

**Definition (free text, please give PubMed ID)**

PMID:20538709

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Better: non-visualisation of the fetal stomach (remove word 'persistent')
and yes, synonyms

Mention that these synonyms are not recommended -- Absent stomach, Absent fetal stomach, Collapsed fetal stomach
